src/pages/product.jsx

```diff
@@ -10,7 +10,7 @@ const Product = () => {
             <ProductPath />
             <ProductInfo />
             <Description />
-            <Products title={'Related Products'} />
+            <Products title={'Related Products'}/>
```

Comment:
![image](https://github.com/zumbost1k/furniro/assets/16866432/3e39f0cf-fc64-4c22-a307-3dd53e53493b)
make you mind, man. there should be a space before closing tag